README.md

```diff
@@ -71,34 +73,52 @@ Some of the rules supported by the makefile:
 | manifests | Generate manifests e.g. CRD, RBAC etc. |
 | generate | Generate the API type definitions. Must be run after modifying the DevfileRegistry type. |
 | bundle | Generate bundle manifests and metadata, then validate generated files. |
-| test_integration | Run the integration tests for the operator. |
+| test-integration | Run the integration tests for the operator. |
 | test | Run the unit tests for the operator. |
+| fmt | Check code formatting |
+| fmt_license | Ensure license header is set on all files |
+| vet | Check suspicious constructs into code |
+| gosec | Check for security problems in non-test source files |
 
 To see all rules supported by the makefile, run `make help`
 
 ## Testing
 
 To run integration tests for the operator, run `make test-integration`. 
 
-By default, the tests will use the default image for the operator, `quay.io/devfile/registry-operator:next`. To use your own image, run:
+The `oc` executable must be accessible.
```

Comment:
In the current state this is required, however we will need to open an issue to allow `kubectl` to be used for the integration test suite given we support OpenShift and Kubernetes.